ios/RNCCameraRoll.mm

```diff
@@ -472,18 +479,13 @@ - (NSDictionary*) convertAssetToDictionary:(PHAsset*)asset
         }
         return;
       }
-      NSString *_Nullable originalFilename = NULL;
-      NSString *_Nullable fileExtension = NULL;
       PHAssetResource *_Nullable resource = NULL;
-      NSNumber* fileSize = [NSNumber numberWithInt:0];
 
-      if (includeFilename || includeFileSize || [mimeTypes count] > 0) {
+      if ([mimeTypes count] > 0) {
         // Get underlying resources of an asset - this includes files as well as details about edited PHAssets
-        // This is required for the filename and mimeType filtering
+        // This is required for mimeType filtering
         NSArray<PHAssetResource *> *const assetResources = [PHAssetResource assetResourcesForAsset:asset];
         resource = [assetResources firstObject];
```

Comment:
These lines are now duplicated because we need `resource` here for mimeType filtering and in  convertAssetToDictionary. I do no know if it is the best way to do. Happy to hear feedback here.